### ISSUE TITLE
[BUG] Incorrect example for java resttemplate client

### ISSUE THREAD

**Issue by RiseOfDeath** [OpenAPITools/openapi-generator#13527]
##### Description

Code from usage guide (which also included into generated client) are not work - it uses class Configuration which not generated by this generator (and some others, f.e. webclient).

##### openapi-generator version

I have tried versions 6.0.1, 6.2.0

##### Steps to reproduce

Just take this example https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/java/resttemplate and try to impelemnt it step by step as in README.md

**Comment by wing328** [comment]
Yup, the auto-generated doc should be updated accordingly for the resttemplate client.

May I know if you've time to file a PR to fix it? I can show you some good starting points.

**Comment by RiseOfDeath** [comment]
Maybe starting point will be good idea because I have never investigate how openapi-generator works inside. (As I gues, it contains some template file with placeholders for documentation, so it should not take much time to fix)
